labs/in-dev-container/deploy/k6.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
```

Comment:
Looks like this file is also a dupplicate, `labs/in-cluster/deploy/k6.yaml`. Is that intentional?

Reply:
yes it is , the idea is to have those 2 labs independent, at least for now.